apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -11,6 +11,15 @@
  * @package full-site-editing
  */
 
+/**
+ * Plugin version.
+ *
+ * Can be used in cache keys to invalidate caches on plugin update.
+ *
+ * @var string
+ */
+define( 'FSE_VERSION', '0.2.1' );
```

Comment:
What do you think about `A8C_FSE_VERSION` just in case?

Reply:
I think it may be time for proper namespacing ;)

Reply:
Sure, I'd be up for that given that we agreed upon minimum PHP requirements. 👍